scripts/s3/mixs3io/mix_object_operations.py

```diff
@@ -231,6 +233,11 @@ def execute_validate_run(self, cmd: str) -> None:
 
         :param cmd: s3bench command to be executed with some workload.
         """
+        if self.s3max_retries:
```

Comment:
Please add s3bench lib and these parameters can be handled in lib.

Reply:
This will be taken care by @VikasKumarC while implementing adapter for all tools.